demo/typescript/exercises/client-exercises.ts

```diff
@@ -2,16 +2,13 @@
 import { Client, ClientConfiguration, endpoints } from "fauna";
 import { getSecret } from "../utils";
 
-
 /**
  * Your task - return a client that talks to Fauna's preview endpoint,
  * has a max_conns to Fauna of 10, and has a query timeout of 60 seconds.
  * You can use the imported getSecret() function to retrieve your secret from
  * the FAUNA_SECRET environmental variable.
-*/
-export function constructingClients() {
-
-}
+ */
+export function constructingClients() {}
```

Comment:
Blast.. pre-commit hook ran `pretty-quick` on the demo files.

Reply:
that seems fine.

Reply:
well we can clean this up with some git magic.

happy to hop on a call and help with that.

Reply:
I added a commit to change the formatting back and skipped the commit hook.

Reply:
oh just let it format. That's its job 😆